packages/design-system/src/css/_primitives.scss

```diff
@@ -0,0 +1,254 @@
+// Primitive Colors
+// Primitive colors shouldn't be used directly in components an other UI elements.
+// They can be only mapped to tokens.
+// Tokens should be used instead in components an other UI elements
+
+@mixin primitives {
```

Comment:
please make sure this does not break our beta setup for dark mode.. you can test it in the console
`window.localStorage.setItem('N8N_THEME', 'dark-beta')`